lib/agenda.js

```diff
@@ -125,13 +123,9 @@ function handleLegacyCreateIndex(err, result, self, cb){
     // Looks like a mongo.version < 2.4.x
     err = null;
     self._collection.ensureIndex(
```

Comment:
Going forward in mongodb, ensureIndex is deprecated in favor of createIndex. Is this an appropriate time to also make this change?